lib/plugins/wallet/geth/base.js

```diff
@@ -52,6 +52,23 @@ function isStrictAddress (cryptoCode, toAddress, settings, operatorId) {
 
 function sendCoins (account, tx, settings, operatorId) {
   const { toAddress, cryptoAtoms, cryptoCode } = tx
+  const isErc20Token = coins.utils.isErc20Token(cryptoCode)
+
+  if (isErc20Token) {
+    return generateErc20Tx(toAddress, defaultWallet(account), cryptoAtoms, false, cryptoCode)
+      .then(pify(web3.eth.sendSignedTransaction))
+      .then(txid => {
+        return pify(web3.eth.getTransaction)(txid)
+          .then(tx => {
+            if (!tx) return { txid }
+
+            const fee = new BN(tx.gas).times(new BN(tx.gasPrice)).decimalPlaces(0)
+
+            return { txid, fee }
+          })
+      })
+  }
```

Comment:
This looks like an exact copy of the code below; I think it would be better to call either `generateErc20Tx()` or `generateTx()` and then run this bit of code on the result of either.

```js
return (isErc20Token ? generateErc20Tx : generateTx)(toAddress, ...)
  .then(...)
```